WordPress/WordPressUITests/Utils/XCTest+Extensions.swift

```diff
@@ -26,13 +26,33 @@ extension XCUIElement {
     }
 }
 
-extension XCTest {
-
-    func isIPhone() -> Bool {
-        return UIDevice.current.userInterfaceIdiom == .phone
-    }
+var isIPhone: Bool {
```

Comment:
We had `isIpad` as a global method, and `isIPhone` as an instance method – it shouldn't be an issue to make both globals – if we want to be pedantic, we can add them as extensions on `UIDevice` or something, but I don't think it's worth it.